framework/src/media/audio/audio_manager.c

```diff
@@ -105,6 +118,20 @@ static int get_avail_audio_out_card_id(void);
 static audio_manager_result_t get_active_audio_device_pcm(struct pcm **pcm, audio_device_type_t type);
 static audio_manager_result_t set_audio_volume(audio_device_type_t type, uint8_t volume);
 
+static uint32_t getClosestSampRateUp(uint32_t rate);
+static uint32_t getClosestSampRateDown(uint32_t rate);
+
+static uint32_t g_supported_samprates[MAX_SUPPORTED_SAMPLERATE] = {
+	AUDIO_SAMP_RATE_8K,
+	AUDIO_SAMP_RATE_11K,
+	AUDIO_SAMP_RATE_16K
+	//AUDIO_SAMP_RATE_22K,
```

Comment:
annotated here

Reply:
Yes. This part must be modified.

Reply:
But this information should be come from h/w.

Reply:
yesterday I added long comment in #1563 (https://github.com/Samsung/TizenRT/pull/1563)
You can find how we can get sample info from h/w